root/components/PostParameters.js

```diff
@@ -28,9 +28,12 @@ const PostParameters = ({
 }: PropsT): React.MixedElement => {
   const [expanded, setExpanded] = React.useState(false);
 
-  const sortedParams = Object.entries(params).sort(
-    (a, b) => compareStrings(a[0], b[0]),
-  );
+  const sortedParams:
+    $ReadOnlyArray<[string, string | $ReadOnlyArray<string>]> =
```

Comment:
Remind me again, what did this type mean here? (sorry, I'm pretty sure I've asked this before)

Reply:
An array of tuples. `Object.entries` returns an array of `[key, value]` items, so in this case the key is `string` and the value is `string | $ReadOnlyArray<string>`.